source/nijigenerate/viewport/common/mesheditor/tools/point.d

```diff
@@ -433,4 +475,25 @@ class ToolInfoImpl(T: PointTool) : ToolInfoBase!(T) {
     string icon() { return ""; }
     override
     string description() { return _("Vertex Tool"); }
+
+    override
+    bool displayToolOptions(bool deformOnly, VertexToolMode toolMode, IncMeshEditorOne[Node] editors) { 
+        if (deformOnly) {
+
+        } else {
+            auto pointTool = cast(PointTool)(editors.length == 0 ? null: editors.values()[0].getTool());
+            igBeginGroup();
+                if (incButtonColored("", ImVec2(0, 0), (pointTool !is null && !pointTool.isAutoConnect())? colorUndefined : ImVec4(0.6, 0.6, 0.6, 1))) {
```

Comment:
I think text color should be active when auto-connect is turned on.
(Currently it looks like option is inactive when auto-connect is turned on.)